Resources/views/CRUD/edit_phpcr_one_association_script.html.twig

```diff
@@ -49,6 +49,7 @@ This code manage the one-to-many association field popup
                 }
                 jQuery('#sonata-ba-field-container-{{ id }}').trigger('sonata.add_element');
                 jQuery('#field_container_{{ id }}').trigger('sonata.add_element');
+                Admin.setup_icheck(jQuery('#field_container_{{ id }}'));
```

Comment:
`jQuery('#field_container_{{ id }}')` is already computed at the previous line, maybe reuse it?